release-notes/aws/10.1.0.md

```diff
@@ -0,0 +1,40 @@
+## :zap:  Giant Swarm Release 10.1.0 for AWS is now active for you! :zap:
+
+We are pleased to announce that this new branch (10.1.0) contains the much anticipated Node Pools release and will contain all
+other releases thereafter such as High Availability of Masters and Spot Instances which are due to be released next year. 
```

Comment:
this is a release, calling the features "releases" is somehow confusing, ok to call them just "features"?

Reply:
good point.. will change to 'features'